oic.r.acl2.raml

```diff
@@ -172,14 +161,12 @@ traits:
                 },
                 "resources": [
                   {
-                    "href": "coaps://IP-ADDR/light",
-                    "rel": "some-rel",
+                    "href": "/light",
                     "rt": ["oic.r.light"],
                     "if": ["oic.if.a"]
                   },
                   {
-                    "href": "coaps://IP-ADDR/door",
-                    "rel": "some-rel",
+                    "href": "/door",
                     "rt": ["oic.r.door"],
                     "if": ["oic.if.a"]
                   }
```

Comment:
A few lines down were we define DELETE, should that be expanded to comply with CR 1485 similar to how its listed in the cred RAML?